package/thingino-esphome/plugins/thingino_media_player/thingino_media_player.c

```diff
@@ -690,17 +714,36 @@ static float get_system_volume(void) {
 // Audio Streaming Functions
 // =============================================================================
 
+#define WAV_HEADER_SIZE	78
```

Comment:
What's this? I think it's probably correct I just want to know why you added this.